bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/QueryablePersistenceService.java

```diff
@@ -28,20 +28,19 @@
 public interface QueryablePersistenceService extends PersistenceService {
 
     /**
-     * Queries the {@link PersistenceService} for data with a given filter criteria
+     * Queries the {@link PersistenceService} for historic data with a given {@link FilterCriteria}.
      *
      * @param filter the filter to apply to the query
      * @return a time series of items
      */
     Iterable<HistoricItem> query(FilterCriteria filter);
 
     /**
-     * Returns a list of items that are stored in the persistence service
+     * Returns a set of {@link PersistenceItemInfo} about items that are stored in the persistence service. This allows
```

Comment:
Do you like a separate PR for this?

Reply:
Might make it clearer that it is completely unrelated to this one here...

Reply:
Done: https://github.com/openhab/openhab-core/pull/1552